bananas/__init__.py

```diff
@@ -1,4 +1,4 @@
-VERSION = (1, 0, 12, 'beta', 1)
+VERSION = (1, 0, 12, 'beta', 2)
```

Comment:
Also, the branch name is getting quite optimistic on when it's going to get merged. You never know 😃 We usually use something like `fix/renaming-reverse-dj19`. Now it's not possible to change so it's okay 😃